hiding-in-the-clouds_secret-management-with-openstack-barbican/heat-templates/master/README.md

```diff
@@ -0,0 +1,20 @@
+# Heat templates for stable/neutron
```

Comment:
Can you rebase this file on `heat-templates/mitaka/README.md` from current master (I clarified that one a bit today)? Thank you!